src/pages/AdminConsole.js

```diff
@@ -214,21 +214,7 @@ class AdminConsole extends Component {
                   iconName: 'manage-dac',
                   iconSize: 'large',
                 })
-              ]),
-              div({
-                // TODO: We won't need this box when MDV is complete
-                isRendered: (this.state.env !== 'dev'),
-                className: 'col-lg-6 col-md-6 col-sm-12 col-xs-12 admin-box' }, [
-                AdminConsoleBox({
-                  id: 'btn_manageDarCollections',
-                  url: '/admin_manage_dar_collections',
-                  color: 'access',
-                  title: 'Manage DAR Collection',
-                  description: 'Select and access DAR Collections for review',
-                  iconName: 'manage-access',
-                  iconSize: 'large',
-                })
-              ]),
```

Comment:
Don't we still ned this for prod?

Reply:
For example, the "Manage DAR Collection" button is visible on prod, but not with this PR:

![Screen Shot 2022-06-29 at 5 14 44 PM](https://user-images.githubusercontent.com/116679/176546117-f082573b-659e-432f-81cc-b3cd8f6930b3.png)

Reply:
I thought we only needed it on dev (soon to be staging) because that's where collections are going to be enabled. Production does not have collections enabled.

Reply:
All environments have collections enabled - it's just the views that are tweaked.